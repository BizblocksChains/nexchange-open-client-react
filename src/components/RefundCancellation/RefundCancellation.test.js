import React from 'react';
import { shallow } from 'enzyme';
import RefundCancellation from './RefundCancellation';

describe('RefundCancellation', () => {
  it('renders correctly', () => {
    const wrapShallow = shallow(<RefundCancellation />).dive();
    expect(wrapShallow).toMatchSnapshot();
  });
});
