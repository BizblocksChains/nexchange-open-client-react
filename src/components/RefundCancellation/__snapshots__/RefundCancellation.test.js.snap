// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RefundCancellation renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <I18n
    ns="translations"
>
    [Function]
</I18n>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="row"
>
        <div
                className="col-xs-12"
        >
                <h2>
                        refund.title
                </h2>
                <p>
                        refund.1
                </p>
                <p>
                        refund.2
                </p>
                <p>
                        refund.3
                </p>
                <p>
                        refund.4
                         
                        <a
                                href="mailto:support@n.exchange"
                        >
                                support@n.exchange
                        </a>
                </p>
        </div>
</div>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="col-xs-12"
>
          <h2>
                    refund.title
          </h2>
          <p>
                    refund.1
          </p>
          <p>
                    refund.2
          </p>
          <p>
                    refund.3
          </p>
          <p>
                    refund.4
                     
                    <a
                              href="mailto:support@n.exchange"
                    >
                              support@n.exchange
                    </a>
          </p>
</div>,
        "className": "row",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2>
              refund.title
</h2>,
            <p>
              refund.1
</p>,
            <p>
              refund.2
</p>,
            <p>
              refund.3
</p>,
            <p>
              refund.4
               
              <a
                            href="mailto:support@n.exchange"
              >
                            support@n.exchange
              </a>
</p>,
          ],
          "className": "col-xs-12",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "refund.title",
            },
            "ref": null,
            "rendered": "refund.title",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "refund.1",
            },
            "ref": null,
            "rendered": "refund.1",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "refund.2",
            },
            "ref": null,
            "rendered": "refund.2",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "refund.3",
            },
            "ref": null,
            "rendered": "refund.3",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "refund.4",
                " ",
                <a
                  href="mailto:support@n.exchange"
>
                  support@n.exchange
</a>,
              ],
            },
            "ref": null,
            "rendered": Array [
              "refund.4",
              " ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "support@n.exchange",
                  "href": "mailto:support@n.exchange",
                },
                "ref": null,
                "rendered": "support@n.exchange",
                "type": "a",
              },
            ],
            "type": "p",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
>
          <div
                    className="col-xs-12"
          >
                    <h2>
                              refund.title
                    </h2>
                    <p>
                              refund.1
                    </p>
                    <p>
                              refund.2
                    </p>
                    <p>
                              refund.3
                    </p>
                    <p>
                              refund.4
                               
                              <a
                                        href="mailto:support@n.exchange"
                              >
                                        support@n.exchange
                              </a>
                    </p>
          </div>
</div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="col-xs-12"
>
            <h2>
                        refund.title
            </h2>
            <p>
                        refund.1
            </p>
            <p>
                        refund.2
            </p>
            <p>
                        refund.3
            </p>
            <p>
                        refund.4
                         
                        <a
                                    href="mailto:support@n.exchange"
                        >
                                    support@n.exchange
                        </a>
            </p>
</div>,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                refund.title
</h2>,
              <p>
                refund.1
</p>,
              <p>
                refund.2
</p>,
              <p>
                refund.3
</p>,
              <p>
                refund.4
                 
                <a
                                href="mailto:support@n.exchange"
                >
                                support@n.exchange
                </a>
</p>,
            ],
            "className": "col-xs-12",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "refund.title",
              },
              "ref": null,
              "rendered": "refund.title",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "refund.1",
              },
              "ref": null,
              "rendered": "refund.1",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "refund.2",
              },
              "ref": null,
              "rendered": "refund.2",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "refund.3",
              },
              "ref": null,
              "rendered": "refund.3",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "refund.4",
                  " ",
                  <a
                    href="mailto:support@n.exchange"
>
                    support@n.exchange
</a>,
                ],
              },
              "ref": null,
              "rendered": Array [
                "refund.4",
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "support@n.exchange",
                    "href": "mailto:support@n.exchange",
                  },
                  "ref": null,
                  "rendered": "support@n.exchange",
                  "type": "a",
                },
              ],
              "type": "p",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
